Cargo.toml

```diff
@@ -1795,6 +1795,16 @@ description = "Demonstrates how the AlignItems and JustifyContent properties can
 category = "UI (User Interface)"
 wasm = false
 
+[[example]]
+name = "grid"
+path = "examples/ui/grid.rs"
+
+[package.metadata.example.grid]
+name = "CSS Grid"
+description = "An example for CSS Grid layout"
+category = "UI (User Interface)"
+wasm = false
```

Comment:
is there a reason it wouldn't work in wasm?

Reply:
@mockersf nothing specific to this PR. But this was set on the `flex_layout` example I copied. Is there an easy way to test running in wasm?

Reply:
wasm works fine. It also works fine for the `flex_layout` example, so I've pushed a commit setting the `wasm` property to true for both examples.